hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azure/StorageInterfaceImpl.java

```diff
@@ -435,6 +456,24 @@ public void startCopyFromBlob(CloudBlobWrapper sourceBlob, BlobRequestOptions op
           null, dstAccessCondition, options, opContext);
     }
 
+    @Override
+    public void startCopyFromBlob(CloudBlobWrapper sourceBlob, BlobRequestOptions options,
```

Comment:
lets define what etag is this. Rename of as dstEtag would be good.

Reply:
Done